ui/labels.js

```diff
@@ -0,0 +1,62 @@
+/*!
+ * jQuery UI Labels @VERSION
+ * http://jqueryui.com
+ *
+ * Copyright jQuery Foundation and other contributors
+ * Released under the MIT license.
+ * http://jquery.org/license
+ */
+
+//>>label: labels
+//>>group: Core
+//>>description: Find all the labels associated with a given input
+//>>docs: http://api.jqueryui.com/labels/
```

Comment:
I've filed https://github.com/jquery/api.jqueryui.com/issues/278 for this. Is the "ponyfill for `input.labels`" description accurate enough?